livox_ros_driver/livox_ros_driver/lddc.h

```diff
@@ -121,6 +121,8 @@ class Lddc {
   ros::NodeHandle *private_node_;
   sensor_msgs::Imu imu_data_;
   rosbag::Bag *bag_;
+
+  const double kGravity_{ 9.80665 };
```

Comment:
this could be made constexpr.

```suggestion
  constexpr double kGravity_ = 9.80665;
```

Reply:
I left the curly brackets initialisation because that's what the code uses (and is more modern than the `=` assignment).